Swat/SwatImageCellRenderer.php

```diff
@@ -134,15 +128,15 @@ public function render()
         $margin_y = 0;
 
         if (
-            $this->occupy_width !== null &&
-            $this->occupy_width > $this->width
+            $this->occupy_width !== null
+            && $this->occupy_width > $this->width
         ) {
             $margin_x = intval($this->occupy_width - $this->width);
```

Comment:
I think the original intention was to handle the case where one or both values included a fractional component. There are probably better ways to avoid that now. I think any fix should be separate from this PR.